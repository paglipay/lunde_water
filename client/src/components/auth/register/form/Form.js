import React, { Component } from 'react';
import { Label, Input } from 'semantic-ui-react';

const Form = () => {
  return (
    <div>
      <div className='field'>
        <label>Email</label>
        <Input type='text' placeholder='Email'/>
        <div className='field'>
          <label>Username</label>
          <Input type='text' placeholder='Username'/>
          <div className='field'>
            <label>Password</label>
            <Input type='text' placeholder='Password'/>
          </div>
          <div className='field'>
            <label>Repeat Password</label>
            <Input type='text' placeholder='Repeat Password'/>
          </div>
        </div>
      </div>
    </div>
  );
};

export default Form;
