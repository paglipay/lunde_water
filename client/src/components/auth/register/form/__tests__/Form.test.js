import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Form from '../Form';

describe('<Form/>', () => {
    it('should render a form with four fields without crashing', () => {
        render(<Form/>);
        const tree = renderer.create(<Form/>).toJSON();

        expect(tree).toMatchSnapshot();
    });

});