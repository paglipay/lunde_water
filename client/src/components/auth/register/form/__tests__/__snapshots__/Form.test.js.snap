// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Form/> should render a form with four fields without crashing 1`] = `
<div>
  <div
    className="field"
  >
    <label>
      Email
    </label>
    <div
      className="ui input"
    >
      <input
        onChange={[Function]}
        placeholder="Email"
        type="text"
      />
    </div>
    <div
      className="field"
    >
      <label>
        Username
      </label>
      <div
        className="ui input"
      >
        <input
          onChange={[Function]}
          placeholder="Username"
          type="text"
        />
      </div>
      <div
        className="field"
      >
        <label>
          Password
        </label>
        <div
          className="ui input"
        >
          <input
            onChange={[Function]}
            placeholder="Password"
            type="text"
          />
        </div>
      </div>
      <div
        className="field"
      >
        <label>
          Repeat Password
        </label>
        <div
          className="ui input"
        >
          <input
            onChange={[Function]}
            placeholder="Repeat Password"
            type="text"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
